index.html

```diff
@@ -71,36 +71,798 @@ <h1 class="primary-text">I’m Melkamu Glad to see you!</h1>
+      
+              <p>javaScript</p>
+      
+            </li>
+          </ul>
+          <!-- project call action section -->
+          <button type="submit" class="call-to-action">See Project</button>
+       
+        </div>
+      </section>
+      <!-- project four end -->
+
+      <div class="card-separator"></div>
+ 
+       <!--about me section start here  -->
+    <section class="about-me-container">
+     
+      <div class="intro-about-me">
+     
+        <div class="about-section">
+     
+          <div class="text-container-about text-container-about-desktop">
+     
+            <h1 class="primary-text-about" id="about">About<br>Myself</h1>
+
+            <p class="secondary-text-about">I’m a software developer! I can help you build a product ,
+              feature or website
+
+              Look through some of my work and experience! If you like what you see and have aproject you need
+              coded,
+              don’t hestiate to contact me.
+            </p>
+          </div>
+
+         <h2 class="link-text link-text-desktop">LET’S CONNECT</h2>
+
+          <div>
+            
+            <ul class="social-container social-container-desktop">
+        
+              <li>  <a href="https://github.com/melkamu372"  rel="noopener" target="_blank">
+
+                  <img  src="./assets/images/github.svg" alt="github"></a>
+              </li>
+      
+              <li> <a href="https://www.linkedin.com/in/melkamu372/"  rel="noopener" target="_blank">
+
+                <img class="social linkedin move" src="./assets/images/Linkedin.svg" alt="linkedin"></a>
+              
+              </li>
+              
+              <li>
+                <a href="#" rel="noopener">
+              
+                  <img src="./assets/images/vector2.svg" alt="agenllist"></a>
+              
+                </li>
+      
+                <li><a href="#" rel="noopener" >
+                    <img src="./assets/images/twitter.svg" alt="twitter"> </a>
+                </li>
+                
+                <li> <a href="#" rel="noopener">
+                    <img  src="./assets/images/vector1.svg" alt="medium"></a>
+                </li>
+                
+              </ul>
+          </div>
+
+          <button type="submit" class="call-action-resume call-action-resume-desktop">Get my resume</button>
+        
+        </div>
+
+        <div class="language-section language-section-desktop">
+
+          <div class="language language-desktop">
+
+            <h2>Languages</h2>
+
+            <img class='language-dropdown' src="./assets/icons/union/Union-down.svg" alt="drop-down">
+
+          </div>
+
+          <div class="language-snippets-container language-snippets-container-desktop">
+
+            <ul class="language-snippets language-snippets-desktop ">
+
+              <li><img src="./assets/icons/language/javascript.svg" alt="javascripticon"></li>
+
+              <li>
+                <p>JavaScript</p>
```

Comment:
To have a codebase that is really semantic, I recommend using a `ul` for each of the language, framework and technology sections, and each and every item is wrapped in their own `ul`.

Like this:

```
<ul>
  <li>
     <h4>framework</h4>
     <ul>
         <li>react</li>
    </ul>
  </li>
</ul>
```